resources/metadata/cloudFoundryDeploy.yaml

```diff
@@ -14,6 +14,11 @@ spec:
       - name: dockerCredentialsId
         description: Jenkins 'Username with password' credentials ID containing user and password to authenticate to the Docker registry.
         type: jenkins
+    resources:
+      - name: deployDescriptor
```

Comment:
I am wondering how this works in the general purpose pipeline. Don't you need e.g. the buildResult?

Reply:
How does it interfere with cloud sdk pipeline stage stashing?

Reply:
I am not sure how it works in the general purpose pipeline, but since it is configured like that for the groovy step currently, it should not create problems there.

Reply:
> How does it interfere with cloud sdk pipeline stage stashing?

That's a good point I will validate if it still works with a test project.

Reply:
It seems to have no impact on Cloud SDK Pipeline use cases.